native/android/app/src/main/res/values/styles.xml

```diff
@@ -1,6 +1,8 @@
 <resources>
-    <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
         <item name="android:windowExitAnimation">@android:anim/fade_out</item>
+        <item name="android:textColor">#000000</item>
+        <item name="android:forceDarkAllowed">false</item>
     </style>
```

Comment:
This sets the default text color to black (changing the text color dynamically is still possible oc). It also disables (not working) support for dark themes.